admin_guide/table_management.rst

```diff
@@ -1,6 +1,54 @@
 Table Management
 $$$$$$$$$$$$$$$$$$
 
+.. _table_size:
+
+Determining Table and Relation Size
+###################################
+
+The usual way to find table sizes in PostgreSQL, :code:`pg_total_relation_size`, drastically under-reports the size of distributed tables. All this function does on a Citus cluster is reveal the size of tables on the coordinator node. In reality the data in distributed tables lives on the worker nodes (in shards), not on the coordinator. A true measure of distributed table size is obtained as a sum of shard sizes. Citus provides helper functions to query this information.
+
++------------------------------------------+----------------------------------------------------------+
+| UDF                                      | Returns                                                  |
++==========================================+==========================================================+
+| citus_relation_size(relation_name)       | Size of actual data in table (the "main fork").          |
+|                                          |                                                          |
+|                                          | A relation can be the name of a table or an index.       |
```

Comment:
This is due to a general style of lists in the docs, I hesitate to adjust the style globally or else other parts of the docs would look different, including the TOC.

Another option is prepending the table with

```rst
.. class:: table-description
```

and mess with CSS to change size and margins.

The third and easiest way would be to use different markup in the right-hand column of this table, just a nested list. That's what I'll go with for now.